### PR TITLE
Add highlevel info about how write leader selection works

### PR DESCRIPTION
**Pull request by PJMODOS**

### CODE REVIEW

File: product_docs/docs/pgd/5/routing/index.mdx [hunk 1]
```diff
@@ -23,6 +23,32 @@ The PGD Proxy connects to one of the EDB Postgres Distributed nodes and monitors
 routing configuration changes as decided by the EDB Postgres Distributed cluster.
 It ensures that the connections are routed to the correct nodes consistently.
 
+The information about currently selected write and read nodes is visible in
+`bdr.node_group_routing_summary`. Note that this is node-local view, the proxy
+always reads from Raft leader to get current and consistent view.
+
+## Leader selection
+
+The write leader is selected by the current Raft leader (either group one or global one,
```
Comment (djw-m) [inline product_docs/docs/pgd/5/routing/index.mdx:32]: local and global? The language previously mandated was sub-group and top-level. The reason for not saying local and global was that the group may not be geo-local and the cluster may not be globally distributed.

```suggestion
The write leader is selected by the current Raft leader (either subgroup one or top-level group one,
```

File: product_docs/docs/pgd/5/routing/index.mdx [hunk 2]
```diff
@@ -23,6 +23,32 @@ The PGD Proxy connects to one of the EDB Postgres Distributed nodes and monitors
...
+depending on whether the leader for local group or global cluster is being selected).
```
Comment (djw-m) [inline product_docs/docs/pgd/5/routing/index.mdx:33]:
```suggestion
depending on whether the leader for sub-group or the cluster's top level group is being selected).
```

File: product_docs/docs/pgd/5/routing/index.mdx [hunk 3]
```diff
@@ -23,6 +23,32 @@ The PGD Proxy connects to one of the EDB Postgres Distributed nodes and monitors
...
+(if enabled) from previous leader. The candidates are ordered by their `route_priority`
+in descending order and by the lag from previous leader in ascending order.
+
+New leader selection process is started when there is either no existing leader currently
+(this could be because there were no valid candidates or because Raft was down), or when
+connectivity is lost to the existing leader.
+
+A Node is considered connected if the last Raft protocol message received from by the leader
+it is not older than Raft election timeout
+(see [Internal settings - Raft timeouts](../pgd-settings#internal-settings---raft-timeouts) ).
```
Comment (djw-m) [inline product_docs/docs/pgd/5/routing/index.mdx:47]: This content is now in the reference section. URL changed.

```suggestion
(see [Internal settings - Raft timeouts](/pgd/latest/reference/pgd-settings#internal-settings---raft-timeouts) ).
```

File: product_docs/docs/pgd/5/routing/index.mdx (hunk 1 above)
Comment (PJMODOS) [reply product_docs/docs/pgd/5/routing/index.mdx:32]: Sure I don't mind, I am used to Kubernetes and TPA calling this global and local

**Comment by PJMODOS** [comment]
@djw-m I wonder if this should go to the 5.5 PR , I edited it through the web interface so it ended up being own PR but...

File: product_docs/docs/pgd/5/routing/index.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/routing/index.mdx:32]: I understand completely.... (and I'd love to switch to local and global for simplicity, but I'll keep this conistent with the other content.

**Comment by djw-m** [comment]
Already backporting it into the reorged 5.5 PGD Proxy docs.

**Comment by PJMODOS** [comment]
> Already backporting it into the reorged 5.5 PGD Proxy docs.

@djw-m Will this go there?

**Comment by djw-m** [comment]
> > Already backporting it into the reorged 5.5 PGD Proxy docs.
> 
> @djw-m Will this go there?

https://deploy-preview-5570--edb-docs-staging.netlify.app/docs/pgd/latest/routing/proxy/#leader-selection

As part of the proxy overview.

**Comment by djw-m** [comment]
As this is now backported. Closing this PR for #5570

**Pull request closed without merging**